packages/web/src/components/Header/_Header.scss

```diff
@@ -0,0 +1,278 @@
+    color: theme.$nav-link-color;
+    background-color: theme.$nav-link-background-color;
+
+    // Current item border
+    &::after {
+        content: '';
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        width: theme.$nav-link-current-border-width;
+        background-color: theme.$nav-link-current-border-color;
+        transform: scaleX(0);
+        transform-origin: right center;
+    }
+
+    &:hover {
+        color: theme.$nav-link-color-hover;
+        background-color: theme.$nav-link-background-color-hover;
+    }
+
+    &:active {
+        color: theme.$nav-link-color-active;
+        background-color: theme.$nav-link-background-color-active;
+    }
+
+    @media (prefers-reduced-motion: no-preference) {
+        // Current item border
+        &::after {
+            transition: transform ease-out 0.25s;
+        }
+    }
+}
+
+.Header__link--current,
+button.Header__link--current {
+    color: theme.$nav-link-color-current;
+    background-color: theme.$nav-link-background-color-current;
+
+    &::after {
+        transform: scaleX(1);
+    }
+}
+
+@include breakpoint.up(theme.$breakpoint) {
+    .Header__bar {
+        position: static;
+        display: flex;
+        flex: 1;
+        align-self: stretch;
+        gap: theme.$header-gap;
+        width: auto;
+        background: transparent;
+        visibility: visible;
+        transform: none;
+        transition: none;
+    }
+
+    .Header__toggle,
+    .Header__bar .Header__close {
+        display: none;
+    }
+
+    .Header__bar .Header__content {
+        display: contents;
+        overflow: visible;
+    }
+
+    .Header__bar .Header__nav {
+        display: flex;
+        align-self: stretch;
+        margin-inline: 0;
+    }
+
+    .Header__bar .Header__actions {
+        flex-direction: row;
+        align-items: center;
+        gap: theme.$actions-column-gap;
+        padding: 0;
+    }
+
+    .Header__bar .Header__actions--secondary {
+        flex: none;
+        margin-left: auto;
+    }
+
+    .Header__bar .Header__link,
+    .Header__bar button.Header__link {
+        display: flex;
+        align-items: center;
+        padding: theme.$nav-link-padding-y-desktop theme.$nav-link-padding-x-desktop;
+
+        // Current item border
+        &::after {
+            top: auto;
+            left: 0;
+            width: auto;
+            height: theme.$nav-link-current-border-width;
+            transform: scaleY(0);
+            transform-origin: bottom center;
+        }
+    }
+
+    .Header__bar .Header__link--current::after,
+    .Header__bar button.Header__link--current::after {
+        transform: scaleY(1);
+    }
+}
+
+.Header--simple {
+    height: 60px;
+    justify-content: center;
+}
+
+.Header--inverted {
+    color: theme.$header-inverted-color;
+    background-color: theme.$header-inverted-background-color;
+}
+
+.Header__bar.is-open {
+    @include breakpoint.down(theme.$breakpoint) {
```

Comment:
I tend to not use max-width media and prefer to reset values in the up direction. The reason is to avoid misuse of the max-width media queries and keep everything mobile-first. But yeah, this is shorter.